.github/workflows/ci.yml

```diff
@@ -0,0 +1,71 @@
+name: Continous Integration
+on:
+  push:
+    branches: [master]
+  pull_request:
+    # The branches below must be a subset of the branches above
+    branches: [master]
+
+env:
+  CI: true
+
+jobs:
+  build:
+    runs-on: ubuntu-18.04
+    strategy:
+      matrix:
+        # See supported Node.js release schedule at https://nodejs.org/en/about/releases/
+        node-version: [12, 14, 16]
+    steps:
+      - uses: actions/checkout@v2
+      - name: Use Node.js ${{ matrix.node-version }}
+        uses: actions/setup-node@v2
+        with:
+          node-version: ${{ matrix.node-version }}
+      - uses: ankane/setup-mysql@v1
+        with:
+          mysql-version: 5.7
```

Comment:
I guess this relates tu ubuntu 18.04 right?, because  I have seen version 8.0 being adopted.

Reply:
yes. It seems like our current code only works with mysql 5.7 and there're some issues with `ubuntu-latest` (i.e. ubuntu-20.04) + mysql 5.7 combination. This is one of the threads I've read: https://ovirium.com/blog/how-to-make-mysql-work-in-your-github-actions/.